# Like the built in NavSatFix except with depth instead of altitude

float64 latitude
float64 longitude
float64 depth      # depth measured positively from the water surface
